app/notifiers/course/assessment_notifier.rb

```diff
@@ -5,4 +5,15 @@ def assessment_attempted(user, assessment)
       notify(assessment.tab.category.course, :feed).
       save
   end
+
+  # To be called when user submitted an assessment.
+  def assessment_submitted(user, course_user, submission)
+    group_managers = course_user.groups.flat_map { |g| g.group_users.manager }.map(&:course_user)
```

Comment:
Can be merged with the tutor finding service maybe, so there's only one place we define the  tutor logic.  @custode You can add a TODO first, I will refactor next, the service also need to be renamed.